metron-platform/metron-test-utilities/src/main/java/org/apache/metron/test/mock/MockHTable.java

```diff
@@ -70,7 +93,7 @@ public static void clear() {
   private final String tableName;
   private final List<String> columnFamilies = new ArrayList<>();
   private HColumnDescriptor[] descriptors;
-
+  private List<Put> putLog;
```

Comment:
@justinleet Do you think it is worth it to try `synchronizedList` again?  IMO, that code is just in a test class, so I am not too concerned.  It also takes a lot of effort to test this.  But if you think it is worth it, I could be convinced.